solidity-v1/dashboard/src/pages/coverage-pools/CoveragePoolPage.jsx

```diff
@@ -168,8 +169,28 @@ const CoveragePoolPage = () => {
               )}
               amountClassName={"h3 text-grey-40"}
               symbolClassName={"h3 text-grey-40"}
+              wrapperClassName={"mb-2"}
             />
           </OnlyIf>
+          <span className={"text-grey-60 flex row"}>
```

Comment:
Do we need `flex row` classess? `span` is an inline element so maybe we can get rid of `flex row`?

Reply:
I think we need this to proper show the icon and the text. I changed the `span` to `div` tho - eea3dbc72d8721a8e9d2819415fd31150b03fbbc